client/my-sites/stats/stats-page-placeholder/index.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+
+const StatsPagePlaceholder = () => {
+	return (
+		<div className="main is-wide-layout">
```

Comment:
Yeah I know :( , these are legacy stats classnames. They could be replaced later.